### PR TITLE
Adding messsage encoding to onboarding FAQ

### PR DESCRIPTION
**Pull request by pirapira**
It took some questions and answers for me to figure out how the messages
are encoded (https://github.com/raiden-network/spec/issues/133 ).

This commit adds what I learned in the FAQ, namely, how to figure out
the format of a message.

### CODE REVIEW

File: docs/onboarding.rst [hunk 1]
```diff
@@ -307,3 +307,6 @@ Frequently Asked Questions
 
 What is MS/PFS?
   These are smart contracts to be deployed in the future.  See `Monitoring Service <https://raiden-network-specification.readthedocs.io/en/latest/terminology.html#term-monitoring-service>`_ and `Pathfinding Service <https://raiden-network-specification.readthedocs.io/en/latest/terminology.html#term-pathfinding-service>`_ in the smart contract specification.
+
+How are messages are encoded?
```
Comment (LefterisJP) [inline docs/onboarding.rst]: remove the second `are`

File: docs/onboarding.rst [hunk 2]
```diff
@@ -307,3 +307,6 @@ Frequently Asked Questions
...
+  The `messages specification <https://raiden-network-specification.readthedocs.io/en/latest/messaging.html>`_ should describe the message formats.  On the implementation, `messages file <https://github.com/raiden-network/raiden/blob/master/raiden/messages.py>`_ contains the details.  See ``encode()`` method of ``Message`` class, and ``packed()`` method of each subclass of ``Message``.
```
Comment (LefterisJP) [inline docs/onboarding.rst]: Please use a commit specific link and not point to master.

Comment (LefterisJP) [inline docs/onboarding.rst]: On the implementation  -> On the implementation side the

File: docs/onboarding.rst (hunk 1 above)
Comment (pirapira) [reply docs/onboarding.rst]: done.

File: docs/onboarding.rst (hunk 2 above)
Comment (pirapira) [reply docs/onboarding.rst]: Replaced it with the current commit ID.

Comment (pirapira) [reply docs/onboarding.rst]: I did something like that.

**Pull request merged**